src/main/webapp/einstellungen.speichern.jsp

```diff
@@ -69,7 +69,7 @@
       benutzer.setLogin(request.getParameter("Benutzername"));
       benutzer.setVorname(request.getParameter("Vorname"));
       benutzer.setNachname(request.getParameter("Nachname"));
-      benutzer.setAdmin(request.getParameter("Administrator").equals("on"));
+      benutzer.setAdmin("on".equals(request.getParameter("Administrator")));
```

Comment:
Does changing this line really make a difference?

Reply:
yes it does !!!  this way it does not get a Null Pointer Exception if request.getParamerter("Administrator") is null and than you make .equalts("on") gives you a Exception the otherway around not

Reply:
Interesting explanation, thanks!